airflow/utils/db.py

```diff
@@ -33,9 +36,7 @@
 # noinspection PyUnresolvedReferences
 from airflow.models.serialized_dag import SerializedDagModel  # noqa: F401  # pylint: disable=unused-import
 # TODO: remove create_session once we decide to break backward compatibility
-from airflow.utils.session import (  # noqa: F401 # pylint: disable=unused-import
-    create_session, provide_session,
```

Comment:
This is here because it is part of the API consumed by our users. It needs to stay. (that's why it has the noqa)